killerbee/scapy_extensions.py

```diff
@@ -10,6 +10,7 @@
 from scapy.base_classes import SetGen
 from scapy.packet import Gen, Raw
 from scapy.all import *
+from scapy import plist
```

Comment:
Good fix!

Reply:
Issue already reported and fixed at https://github.com/riverloopsec/killerbee/issues/207